codedog/actors/reporters/code_review.py

```
@@ -1,3 +1,7 @@
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)